### PR TITLE
New Human Species Datum

### PR DESCRIPTION
**Pull request by kilkun**
Adds in corporate cyborg datum
Cybernetic post-human inspired by Bullfrog's Syndicate
-High melee damage
-High damage resistances
-Custom sprite
-Slightly tweaked datum from deathsquad PR to be slightly less tough.

A small, unused (for now) species that will later be incorporated into it's own gamemode when I understand gamemode functions and methods at a later date. For now it's useless though.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
pictures

**Comment by KazeEspada** [comment]
Kilkun, it seems like a travis error.

**Comment by kilkun** [comment]
sheeyit. I'll fix this in the morning when I'm not typing through a bottle of wine.

**Comment by Alek2ander** [comment]
Objective #1: I need your clothes...
Objective #2: Your boots...
Objective #3: And your Escape Shuttle.

**Comment by kilkun** [comment]
Agent Sprites

The modified human sprites. Also should fix travis.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -720,3 +720,31 @@ var/global/list/synth_flesh_disguises = list()
...
 		return ..()
+
+
+/*
+SYNDICATE BLACK OPS
+*/
+//The hardcore return of the failed Deathsquad augmentation project
+//Now it's own, wizard-tier, very robust, lone antag
+/datum/species/corporate
+ name = "Corporate Agent"
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species_types.dm]: the indentation on this is all wrong.
for every var on the datum.

**Comment by kilkun** [comment]
Whew. Too used to using spaces instead of tabs in GML, forgot that DML is reliant on indentation.

**Comment by RemieRichards** [comment]
I've used GML once.

I wrote an entire resource system for my friend who /loves GML a bit too much/ because he's a moron.
I didn't even know GML at the time (still don't) but he still uses that system lol.

**Comment by Niknakflak** [comment]
Those sprites look like you grayscaled half the head and thats it.  

:-1: needs something more

**Comment by kilkun** [comment]
GML's great because it's got a simple syntax and the IDE is designed around lightspeed prototyping.

Got it for 5 bux in that indie bundle and the only thing I don't like is that classes are dumb to implement and methods basically don't exist outside of their preset ones.

@Niknakflak
It's the same sprite I used for the Deathsquad PR.
Syndicate Agent
It's also what the inspiration is. It'll look better in action, for reasons yet to be disclosed, it's not really viable to strip an agent even if you knock one out.

**Comment by KazeEspada** [comment]
Where are all the colors?

**Comment by kilkun** [comment]
Trenchcoats and masks were dark grey in the actual game. It's blue in that picture because early 90's video game art.

**Comment by Dorsisdwarf** [comment]
Speedmod: 2 is faster? 

Did someone really design a system in which low speeds were fast, but low speedmods were slow?

**Comment by Niknakflak** [comment]
What the sprites are supposed to be make look cool but I'm just saying it's pretty boring because no one will really see them looking any different in game

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -720,3 +720,31 @@ var/global/list/synth_flesh_disguises = list()
...
+*/
+//The hardcore return of the failed Deathsquad augmentation project
+//Now it's own, wizard-tier, very robust, lone antag
+/datum/species/corporate
+	name = "Corporate Agent"
+	id = "agent"
+	hair_alpha = 0
+	need_nutrition = 0//They don't need to eat
+	say_mod = "declares"
+	speedmod = 2//Fast
```
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/species_types.dm]: WHAT THIS EXISTS

I CAN FINALLY FINISH MY HEDGEHOG-MAN PR

Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/species_types.dm]: speedmod is backwards, should be -2.

**Comment by KazeEspada** [comment]
And by colors I mean, there are flags you need to set to make sure they have eye and colors and stuff.

**Comment by KorPhaeron** [comment]
@kilkun

Fix the speed thing

**Comment by kilkun** [comment]
@KazeEspada Eye color is randgen, there's only one skintype, hair is invisible. Not sure what you mean by color flags, but from my testing they looked fine.

**Comment by KorPhaeron** [comment]
Interested to see what step 2 is gonna be

**Pull request merged**